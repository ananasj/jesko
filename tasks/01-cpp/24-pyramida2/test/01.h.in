16
9 10 9 16 19 8 0 5 11 0 11 13 19 18 11 18
