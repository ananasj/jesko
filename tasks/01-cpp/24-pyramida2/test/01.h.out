19 19 25 35 27 8 5 16 11 11 24 32 37 29 29
38 44 60 62 35 13 21 27 22 35 56 69 66 58
82 104 122 97 48 34 48 49 57 91 125 135 124
186 226 219 145 82 82 97 106 148 216 260 259
412 445 364 227 164 179 203 254 364 476 519
857 809 591 391 343 382 457 618 840 995
1666 1400 982 734 725 839 1075 1458 1835
3066 2382 1716 1459 1564 1914 2533 3293
5448 4098 3175 3023 3478 4447 5826
9546 7273 6198 6501 7925 10273
16819 13471 12699 14426 18198
30290 26170 27125 32624
56460 53295 59749
109755 113044
222799
